src/graphql/v1/resolvers/custom/patient.js

```diff
@@ -78,6 +78,18 @@ module.exports = {
                 'CarePlan'
             );
         },
+        careTeam: async (parent, args, context, info) => {
+            return await getResources(
+                parent,
+                {
+                    ...args,
```

Comment:
This line leads me to believe we can add more filters, but I don't see it exposed in the schema.